src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpSystemProxy.cs

```diff
@@ -135,11 +137,11 @@ private HttpSystemProxy(WinInetProxyHelper proxyHelper, SafeWinHttpHandle sessio
                                             RegexOptions.Compiled | RegexOptions.IgnoreCase | RegexOptions.CultureInvariant);
                             _bypass.Add(re);
                         }
-                        catch
+                        catch (Exception ex)
                         {
                             if (NetEventSource.IsEnabled)
                             {
-                                NetEventSource.Info(this, "Failed to process " + tmp + " from bypass list.");
+                                NetEventSource.Error(this, ex, "Failed to process " + tmp + " from bypass list.");
```

Comment:
I don't think this is calling the Error method you want to call.  It's going to treat the message "Failed..." as the current method name.

Reply:
I just manually tested this. You're right.

Fixed.